src/types.ts

```diff
@@ -12,11 +12,15 @@ export enum Command {
 	CLOSE_MODAL = 'close_modal',
 	GET_SIGNED_TOKEN = 'get_signed_token',
 	REDIRECT_TO = 'redirect_to',
+	SHOW_FLOATING_WINDOW = 'show_floating_window',
+	HIDE_FLOATING_WINDOW = 'hide_floating_window',
+	MINIMIZE_FLOATING_WINDOW = 'minimize_floating_window',
```

Comment:
Is there use case for app to minimize the window? Probably they only need closing and opening.

Maybe the opening and closing of CFW can be combined into single command? Something like `SET_FLOATING_WINDOW_VISIBILITY`

Reply:
It's being here since we didn't know if we're going to have 2 window actions (minimize and close) or just 1 to hide floating window. I'll remove it from here for now since afaik there is no final decision on it.
As for `SET_FLOATING_WINDOW_VISIBILITY`, it is possible and both options probably have the right to exist, although I'd like it to be a separate command for each action.